Library/Homebrew/diagnostic.rb

```diff
@@ -953,67 +702,6 @@ def check_for_bad_python_symlink
         EOS
       end
 
-      def check_for_non_prefixed_coreutils
```

Comment:
I think things are still likely to try to pick these up if they're in /usr/local/bin regardless of env filtering